src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ZombieClusterMonitor.scala

```diff
@@ -95,9 +96,18 @@ class ZombieClusterMonitor(config: ZombieClusterConfig, gdDAO: GoogleDataprocDAO
   }
 
   private def isClusterActiveInGoogle(cluster: Cluster): Future[Boolean] = {
-    // Clusters in Creating status may not yet exist in Google. Therefore treat all Creating clusters as active.
+    // Clusters in Creating status may not yet exist in Google. Clusters in creating (that are not hanging, as defined by config.creationHangTolerance) are considered active
     if (cluster.status == ClusterStatus.Creating) {
-      Future.successful(true)
+
+        val secondsSinceClusterCreation: Long = Duration.between(cluster.auditInfo.createdDate, Instant.now()).getSeconds
+
+        //determine if the creating cluster has been hanging past the tolerance threshold
+        if (secondsSinceClusterCreation > config.creationHangTolerance.toSeconds) {
+           Future.successful(false)
```

Comment:
Should this `if` block check the cluster status in Google? I'm thinking we want the following logic:

| status in Leo | present in Google | isClusterActiveInGoogle returns... |
| ------------- | ------------------| ---------------------------------|
| Creating, inside the hang tolerance | yes | true |
| Creating, inside the hang tolerance | no | true |
| Creating, outside the hang tolerance | yes | true |
| Creating, outside the hang tolerance | no | false |